mmcv/runner/hooks/logger/pavi.py

```diff
@@ -110,6 +125,25 @@ def get_step(self, runner) -> int:
         else:
             return self.get_iter(runner)
 
+    def _add_ckpt(self, runner, last_ckpt=False) -> None:
+        # runner.epoch += 1 has been done before `after_run`.
+        iteration = runner.epoch if self.by_epoch else runner.iter
```

Comment:
Should `self.get_epoch` and `self.get_iter` be used here, since `runner.epoch` counts from 0.

Reply:
Thanks for the reply, all things are fixed, please let me know if there are anything else to change.